hack/jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -0,0 +1,168 @@
+- job:
+    name: kubernetes-pull-test-unit-integration
+    logrotate:
+        daysToKeep: 7
+    properties:
+        - github:
+            url: https://github.com/kubernetes/kubernetes
+        - throttle:
+            max-per-node: 2
+    parameters:
+        - string:
+            name: sha1
+            description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
+    scm:
+        - git:
+            remotes:
+                - remote:
+                    url: https://github.com/kubernetes/kubernetes
+                    refspec: '+refs/heads/*:refs/remotes/upstream/*'
+                - remote:
+                    url: https://github.com/kubernetes/kubernetes
+                    refspec: '+refs/pull/${ghprbPullId}/merge:refs/remotes/origin/pr/${ghprbPullId}/merge'
+            branches:
+                - 'origin/pr/${ghprbPullId}/merge'
+            browser: githubweb
+            browser-url: http://github.com/kubernetes/kubernetes
+            timeout: 20
+            clean:
+                after: true
+            wipe-workspace: false
+            skip-tag: true
+    triggers:
+        - github-pull-request:
+            # This is the Jenkins GitHub ID, not the actual github token.
+            auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            trigger-phrase: '(?is).*@k8s-bot\s+unit\s+test\s+this.*'
+            cron: 'H/2 * * * *'
+            status-context: Jenkins unit/integration
+            status-url: 'http://pr-test.k8s.io/${ghprbPullId}/${JOB_NAME}/${BUILD_NUMBER}/'
+            admin-list:
+                - alex-mohr
```

Comment:
is this everyone in https://github.com/orgs/kubernetes/teams/kubernetes-collaborators? it'd probably be a good time to compare the kubernetes-collaborators list vs. this list and work out any discrepancies.